libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -260,9 +250,12 @@ private Optional<IdentityClaim> generateIdentityClaim(List<VcStoreItem> successf
 
         List<IdentityClaim> identityClaims = new ArrayList<>();
         for (VcStoreItem item : successfulVCStoreItems) {
-            if (EVIDENCE_CRI_TYPES.contains(item.getCredentialIssuer())) {
-                identityClaims.add(
-                        getIdentityClaim(item.getCredential(), item.getCredentialIssuer(), true));
+            JsonNode vcEvidenceNode = getVCNode(item.getCredential(), VC_EVIDENCE);
+            for (JsonNode evidence : vcEvidenceNode) {
+                if (evidence.path(VC_EVIDENCE_VALIDITY).isInt()
+                        && evidence.path(VC_EVIDENCE_STRENGTH).isInt()) {
```

Comment:
Can we extract this out to a method so it's clear what we're trying to do (i.e. `isEvidenceVc`) or similar?

I'm slightly torn as to whether we should also be checking that these are both non-zero - we do check that already via the 'success' check, but I don't love relying on it.

Reply:
At the moment I've opted to check whether they exist/are integers, it's easy enough to check if they are non-zero should we want that change